package.json

```diff
@@ -409,8 +411,10 @@
     "os-name": "^3.1.0",
     "proxyquire": "^2.1.3",
     "ps-list": "^7.1.0",
-    "public-ip": "^4.0.1",
+    "query-string": "^7.0.0",
```

Comment:
Nit: is this dependency needed? Node has a builtin `querystring` module: https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options

Reply:
Oh thank you! Removed the dependency.